src/class_objectProcessor.py

```diff
@@ -1,10 +1,14 @@
+"""
+src/class_objectProcessor.py
+=================================
+"""
 import hashlib
 import random
-import shared
 import threading
 import time
 from binascii import hexlify
 from subprocess import call  # nosec
+import shared
```

Comment:
Please add a blank line

Reply:
Okay. Thanks

Reply:
[PEP8](https://www.python.org/dev/peps/pep-0008/#imports) requires that explicitly:

> You should put a blank line between each group of imports.